### PR TITLE
also consider libfabric dep (--with-ofi) when setting default OpenMPI configure options

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by Micket — APPROVED** [review]
lgtm

File: easybuild/easyblocks/o/openmpi.py [hunk 1]
```diff
@@ -77,11 +77,15 @@ def config_opt_unused(key, enable_opt=False):
                 self.cfg.update('configopts', '--without-verbs')
 
         # handle dependencies
-        for dep in ['CUDA', 'hwloc', 'libevent', 'PMIx', 'UCX']:
+        for dep in ['CUDA', 'hwloc', 'libevent', 'libfabric', 'PMIx', 'UCX']:
             if config_opt_unused(dep.lower()):
                 dep_root = get_software_root(dep)
                 if dep_root:
-                    self.cfg.update('configopts', '--with-%s=%s' % (dep.lower(), dep_root))
+                    if dep in 'libfabric':
```
Comment (Micket) [inline easybuild/easyblocks/o/openmpi.py:84]:
```suggestion
                    if dep == 'libfabric':
```

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyblocks/o/openmpi.py (hunk 1 above)
Comment (boegel) [reply easybuild/easyblocks/o/openmpi.py:84]: Oh nice catch 🤦

**Review by Micket — APPROVED** [review]
lgtm

**Pull request merged**